### ISSUE TITLE
NameError: uninitialized constant Redmine::WikiFormatting::Markdown in redmine version 6.0.1

### ISSUE THREAD

**Issue by mario-spitze** [mikitex70/redmine_drawio#139]
After updating to redmine version 6.0.1 i get an error at startup.

The service is started by docker and the drawio plugin ist updated to version 1.4.9

It seems that there was a breaking change at the WikiFormatting module. -> https://www.redmine.org/issues/41513

Do you need some more information? I attach the log from `docker compose up`

```
[+] Running 1/0
 ✔ Container redmine-redmine-1  Created                                                                                                                                                                                                  0.0s 
Attaching to redmine-1
redmine-1  | Your Gemfile lists the gem puma (>= 0) more than once.
redmine-1  | You should probably keep only one of them.
redmine-1  | Remove any duplicate entries and specify the gem only once.
redmine-1  | While it's not a problem now, it could cause errors if you change the version of one of them later.
redmine-1  | The Gemfile's dependencies are satisfied
redmine-1  | WARN: Unresolved or ambiguous specs during Gem::Specification.reset:
redmine-1  |       stringio (>= 0)
redmine-1  |       Available/installed versions of this gem:
redmine-1  |       - 3.1.2
redmine-1  |       - 3.1.1
redmine-1  | WARN: Clearing out unresolved specs. Try 'gem cleanup <gem>'
redmine-1  | Please report a bug if this causes problems.
redmine-1  | rake aborted!
redmine-1  | NameError: uninitialized constant Redmine::WikiFormatting::Markdown (NameError)
redmine-1  | 
redmine-1  |     module Redmine::WikiFormatting::Markdown::Helper
redmine-1  |                                   ^^^^^^^^^^
redmine-1  | /usr/src/redmine/plugins/redmine_drawio/lib/redmine_drawio/helpers/markdown_helper.rb:50:in `<top (required)>'
redmine-1  | /usr/local/bundle/gems/zeitwerk-2.7.1/lib/zeitwerk/core_ext/kernel.rb:34:in `require'
redmine-1  | /usr/src/redmine/plugins/redmine_drawio/lib/redmine_drawio.rb:13:in `<top (required)>'
redmine-1  | /usr/local/bundle/gems/zeitwerk-2.7.1/lib/zeitwerk/core_ext/kernel.rb:34:in `require'
redmine-1  | /usr/src/redmine/plugins/redmine_drawio/after_init.rb:2:in `<top (required)>'
redmine-1  | /usr/src/redmine/plugins/redmine_drawio/init.rb:23:in `require_relative'
redmine-1  | /usr/src/redmine/plugins/redmine_drawio/init.rb:23:in `<top (required)>'
redmine-1  | /usr/src/redmine/lib/redmine/plugin_loader.rb:31:in `load'
redmine-1  | /usr/src/redmine/lib/redmine/plugin_loader.rb:31:in `run_initializer'
redmine-1  | /usr/src/redmine/lib/redmine/plugin_loader.rb:61:in `each'
redmine-1  | /usr/src/redmine/lib/redmine/plugin_loader.rb:61:in `block in load'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:407:in `instance_exec'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:407:in `block in make_lambda'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:179:in `block in call'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:668:in `block (2 levels) in default_terminator'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:667:in `catch'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:667:in `block in default_terminator'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:180:in `call'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:559:in `block in invoke_before'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:559:in `each'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:559:in `invoke_before'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:109:in `run_callbacks'
redmine-1  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/reloader.rb:96:in `prepare!'
redmine-1  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/application/finisher.rb:74:in `block in <module:Finisher>'
redmine-1  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/initializable.rb:32:in `instance_exec'
redmine-1  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/initializable.rb:32:in `run'
redmine-1  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/initializable.rb:61:in `block in run_initializers'
redmine-1  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/initializable.rb:60:in `run_initializers'
redmine-1  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/application.rb:435:in `initialize!'
redmine-1  | /usr/src/redmine/config/environment.rb:16:in `<top (required)>'
redmine-1  | /usr/local/bundle/gems/zeitwerk-2.7.1/lib/zeitwerk/core_ext/kernel.rb:34:in `require'
redmine-1  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/application.rb:411:in `require_environment!'
redmine-1  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/application.rb:559:in `block in run_tasks_blocks'
redmine-1  | /usr/local/bundle/gems/rake-13.2.1/exe/rake:27:in `<top (required)>'
redmine-1  | Tasks: TOP => db:migrate => db:load_config => environment
redmine-1  | (See full trace by running task with --trace)
redmine-1 exited with code 1
```

**Comment by mario-spitze** [comment]
This seams to be solved in pull #142  ?

**Comment by mikitex70** [comment]
Give me some time to test it, but I'm pretty sure it will solve the problem.
@liaham always does a good job.

**Comment by mikitex70** [comment]
Hi @mario-spitze, just released `v1.5.0` which fixes the issue on Redmine 6.
